### PR TITLE
Upgrade datapusher

### PR DESCRIPTION
**Pull request by avdata99**
- Added a datapusher container using upstream verison.
 - [Fixed and error in CKAN](https://github.com/ckan/ckan/pull/5281/files) who fail to use datapusher in a separated container (temporary using my fork)

Now we can run CKAN 2.8 + datapusher and test them with docker-compose

### CODE REVIEW

File: datapusher/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+FROM keitaro/base:0.4
```
Comment (adborden) [inline datapusher/Dockerfile]: Is there no existing datapusher image? If we are forced to build our own, then we should build our own image from its own repository. We shouldn't be committing a datapusher Dockerfile here.

Comment (avdata99) [reply datapusher/Dockerfile]: It seems you use a local datapusher in the [inventory app](https://github.com/GSA/inventory-app/blob/inventory_ckan_2.8/config/production.ini#L158) for development. Also in [datagov-deploy](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini#L201)
I din't find a docker image in [GSA's Docker hub](https://hub.docker.com/u/datagov).
This repo seems the best place to test the upgrade.
Does GSA have a repository for the datapusher image?

Comment (avdata99) [reply datapusher/Dockerfile]: Not sure but it seems [in datagov-deploy you just run datapusher locally](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/inventory/defaults/main.yml#L11-L24) in the inventory machine.

Comment (avdata99) [reply datapusher/Dockerfile]: If you prefer @adborden we can start a new repo for GSA datapusher to work independently.

**Comment by avdata99** [comment]
Using Xloader could replace datapusher: https://github.com/GSA/inventory-app/pull/29

**Comment by avdata99** [comment]
This PR is closed because we select Datapusher from an image a not with code.

**Pull request closed without merging**